compiler/compiler.go

```diff
@@ -281,9 +281,6 @@ func CompilePackage(moduleName string, pkg *loader.Package, ssaPkg *ssa.Package,
 	c.runtimePkg = ssaPkg.Prog.ImportedPackage("runtime").Pkg
 	c.program = ssaPkg.Prog
 
-	// Convert AST to SSA.
-	ssaPkg.Build()
-
```

Comment:
This should ideally have been removed in #3920 and although it doesn't really harm (packages are only built once), it is better to remove it.